api/v1beta2/network_types.go

```diff
@@ -446,6 +475,45 @@ func (s *SubnetSpec) String() string {
 	return fmt.Sprintf("id=%s/az=%s/public=%v", s.GetResourceID(), s.AvailabilityZone, s.IsPublic)
 }
 
+// IsEdge returns the true when the subnet is created in the edge zone,
+// Local Zones.
+func (s *SubnetSpec) IsEdge() bool {
+	if s.ZoneType == nil {
+		return false
+	}
+	if s.ZoneType.Equal(ZoneType(ZoneTypeLocalZone)) {
+		return true
+	}
+	return false
```

Comment:
```suggestion
    return s.ZoneType != nil && *s.ZoneType == ZoneTypeLocalZone
```

Reply:
Then I wouldn't mind leaving it as it is. I forgot this was broken out from the future changes.